### ISSUE TITLE
Set img

### ISSUE THREAD

**Issue by jalvacastillo** [harryy2510/ngx-img#14]
It's great!

But how I can load an initial default image?

**Comment by ziomyslaw** [comment]
This works for me. 
```
<ngx-img #imageuploader (onSelect)="onSelect($event)" (onReset)="reset()" [config]="config"></ngx-img>

@ViewChild('imageuploader') private imageUploader: NgxImgComponent;

public ngAfterViewInit(): void {
    this.imageUploader.hasPreview = true;
    this.imageUploader.imgSrc = this.initialImageUrl;
  }
```

`@Input imgSrc` should be ok, but template requires `hasPreview = true`. 
```
<ngx-img [imgSrc]="initialImageUrl" (onSelect)="onSelect($event)" (onReset)="reset()" [config]="config"></ngx-img>
```

**Comment by harryy2510** [comment]
Fixed in v10.19.3
